vite.config.ts

```diff
@@ -18,7 +18,7 @@ export default defineConfig({
       fileName: (format) => `index.${format}.js`,
     },
     rollupOptions: {
-      external: ['preact', 'react', 'react-toastify'],
+      external: ['preact', 'react', 'reactToastify', '/__tests__/'],
```

Comment:
![image](https://user-images.githubusercontent.com/49658988/200047166-06893698-b4ef-4d83-809e-34f1b7ecf957.png)
We had this warning, I fixed it